modules/core/src/main/scala/Trace.scala

```diff
@@ -24,6 +24,7 @@ trait Trace[F[_]] {
   /** Create a new span, and within it run the continuation `k`. */
   def span[A](name: String)(k: F[A]): F[A]
 
+  def spanR(name: String): Resource[F, Span[F]]
```

Comment:
The idea with `Trace` is that there is an ambient span, so it would be more consistent to have the resource yield `Unit` I think.

Reply:
That makes it easier... And makes sense, as I can put my stuff into the span by using the method on Trace.